blocks/cards/cards.js

```diff
@@ -1,6 +1,8 @@
 import { createOptimizedPicture, readBlockConfig, toClassName } from '../../scripts/aem.js';
 import { a, div, h5 } from '../../scripts/dom-helpers.js';
-import { FILTERS, fetchAPI, getLocaleBlogs } from '../../scripts/scripts.js';
+import { FILTERS, fetchAPI, getLocaleBlogs, getLocale } from '../../scripts/scripts.js';
+
+const TAGS_QUERY_INDEX ="/blogs/tags.json"
```

Comment:
under the assumption that this will be used in other places, would it make sense to move this to `scripts.js`?

Reply:
Might be used at other places later so moved it to scripts.